apps/andi/mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Andi.MixProject do
   def project do
     [
       app: :andi,
-      version: "2.1.15",
+      version: "2.1.18",
```

Comment:
Typo? Or anticipating version bumps from other PRs?

Reply:
Anticipating the version bumps!